tasks/windows/install-package-chocolatey.yml

```
@@ -1,8 +1,41 @@
 ---
+- name: Get info for the windows_exporter service
+  ansible.windows.win_service_info:
+    name: "windows_exporter"
+  register: service_info
+  ignore_errors: true
```

```suggestion
  ignore_errors: true
  when: windows_exporter_textfile_dir is defined
```

I guess we only care about doing this if that var is defined, right?

---

Good point, that's correct. I added the condition in 19e314ecb009d136e8bf09cd0dc50ae342168d60.